redbot/cogs/audio/audio.py

```diff
@@ -2128,8 +2128,8 @@ def _restart_connect(self):
 
     @commands.command()
     @commands.guild_only()
-    async def seek(self, ctx, seconds: int = 30):
-        """Seek ahead or behind on a track by seconds."""
+    async def seek(self, ctx, seconds):
+        """Seek ahead or behind on a track by seconds or a to a specific time."""
```

Comment:
Might be good to include similar documentation here as #2465 for how to skip to a specific time in the track.